source/features/list-prs-for-file.tsx

```diff
@@ -103,7 +103,7 @@ async function init(): Promise<void> {
 		prs = prs.filter(pr => pr !== Number(editingPRNumber));
 	}
 
-	if (prs.length === 0) {
+	if (!prs || prs.length === 0) {
```

Comment:
Let’s have a single if `!pr, return` on line 100, so we don’t have to test pr twice

Reply:
But prs can be empty after the filter (len of 0)

Reply:
Yeah, `if !prs` before and `prs.length === 0` after. It's still 2 separate `if`s but we don't have to use `pr ||` in 2 conditions

Reply:
Got it